malariagen_data/anoph/base.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         check_location: bool = False,
         storage_options: Optional[Mapping] = None,
         results_cache: Optional[str] = None,
+        tqdm_class=None,
```

Comment:
Exposing this parameter to allow overriding default tqdm behaviour for jupyter lab environments where widgets don't seem to work - can force back to using stdout.